### PR TITLE
Jw/guide redirects

### PR DESCRIPTION
**Pull request by jarrodwatts**

### CODE REVIEW

**Comment by jarrodwatts** [comment]
@nachoiacovino here are changes you suggested in discord

File: vercel.json [hunk 1]
```diff
@@ -204,6 +204,21 @@
...
+    },
+    {
+      "source": "/guides/create-your-own-marketplace-with-thirdweb-typescript-sdk",
+      "destination": "/guides/nft-marketplace-with-typescript-next",
+      "permanent": true
+    },
+    {
+      "source": "/guides/switch-between-networks",
+      "destination": "/react/react.usenetwork",
+      "permanent": true
```
Comment (jnsdls) [inline vercel.json:221]: we cannot use a permanent redirect with a *generated* destination, that destination can change with no knowledge on the docs side

**Review by jnsdls — CHANGES_REQUESTED** [review]
redirects have to be carefully considered also if unsure never use permanent redirect

File: vercel.json (hunk 1 above)
Comment (jarrodwatts) [reply vercel.json]: removed permanent from all three

Comment (nachoiacovino) [reply vercel.json]: First two were fine, the third one was the problem because it was generated

Comment (jarrodwatts) [reply vercel.json]: ok added the two back

**Review by jnsdls — APPROVED** [review]

**Pull request merged**